zoo-css-challenge/index.html

```diff
@@ -1,136 +1,220 @@
+
+        <p>
+          Giraffes use their height to good advantage and browse on leaves and
+          buds in treetops that few other animals can reach (acacias are a
+          favorite). Even the giraffe's tongue is long! The 21-inch tongue helps
+          them pluck tasty morsels from branches. Giraffes eat most of the time
+          and, like cows, regurgitate food and chew it as cud. A giraffe eats
+          hundreds of pounds of leaves each week and must travel miles to find
+          enough food.
+        </p>
+      </section>
+
+      <section class="my-animal">
+        <h1>My favorite animal</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
+          dolorum quibusdam atque ipsum magni accusantium perferendis at sunt!
+        </p>
+
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
+          dolorem iusto dolore, similique optio sit natus pariatur nisi maiores
+          fugit.
+        </p>
+
+        <h2>Lorem ipsum dolor sit.</h2>
+
+        <p>
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Animi,
+          saepe! Maiores laboriosam illum molestiae. Asperiores laboriosam dicta
+          perspiciatis, ut maxime assumenda delectus facilis accusamus, dolorum
+          laborum labore quia. Neque vero ut minima eaque voluptatibus.
+        </p>
+      </section>
+
+      <section class="news">
+        <h2>News</h2>
+        <div class="card-time">
+          <div class="card">
+            <time>2021-05-30</time>
+            <h4>Which big cat are you?</h4>
+            <a>Find out More</a>
+          </div>
+          <div class="card">
+            <time>2021-05-30</time>
+            <h4>Which big cat are you?</h4>
+            <a>Find out More</a>
+          </div>
+
+          <div class="card">
+            <time>2021-05-30</time>
+            <h4>Which big cat are you?</h4>
+            <a>Find out More</a>
+          </div>
+        </div>
+      </section>
+
+      <section class="education">
+        <div class="education-container">
+          <h3>Learning resources for schools</h3>
+          <p>
+            Explore our activities, videos, and lesson plans to support a wide
+            range of curriculum topics
+          </p>
+          <button>Find them here</button>
+        </div>
+      </section>
+
+      <section class="membership">
+        <h2>Are there any benefits or discounts for members?</h2>
+        <p>
+          There are <strong>AMAZING</strong> benefits of becoming a member with
+          Birmingham Zoo! Our members get special discounts in our shops, cafes,
+          and unlimited entry
+        </p>
+        <button>Learn More</button>
+      </section>
+
+      <section class="programs">
+        <h2>Programs</h2>
+        <div class="cards">
+          <div class="card">
+            <h4>Children and Family Programs</h4>
+            <a href="#">Find out More</a>
```

Comment:
Each of these links should have target="_blank" so they open in a new tab

Reply:
I've added the target blank in every link.